Apps/WebClient/src/ClientApp/app/views/timeline.vue

```diff
@@ -226,7 +226,7 @@ export default class TimelineComponent extends Vue {
       .getPatientMedicationStatements(this.user.hdid)
       .then(results => {
         console.log(results);
-        if (!results.errorMessage) {
+          if (results.resultStatus == 0) {

```

Comment:
Instead of 0 we can use an Enum to type it

Reply:
I've made this change.